pymeasure/instruments/fluke/fluke7341.py

```diff
@@ -0,0 +1,68 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2021 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+from pymeasure.adapters import SerialAdapter
+
+class FlukeSerialAdapter(SerialAdapter):
```

Comment:
I chose `SerialAdapter` since it is much more simple than `VisaAdapter` and I felt could be more appropriate for that simple non SCPI instrument. I am not sure this is acceptable. What is your opinion ? On the other hand, I agree with your comment on removing the class `FlukeSerialAdapter` and I replaced it with a re-implementation of `write`.

Reply:
I understand your argument, however, the advantage of the visa adaptor is that the communication is handled by visa; one of the advantages of this is that the having the instrument open in one script doesn't block opening another connection via visa.

Reply:
I have tried to implement it using `VISAAdapter `as follow:
```
    def __init__(self, port):
        adapter = VISAAdapter(port, write_termination='\r\n', timeout=2000, preprocess_reply=lambda x: x.split()[1])
        adapter.connection.baudrate = 2400
        super(Fluke7341, self).__init__(
            adapter,
            "Fluke 7341",
            includeSCPI=False
        )
```
Is this OK, in your opinion ?
Few notes:
- I need to set the `baudrate` explicitly, since `safeKeywords` "feature" (see #334)
- The port parameter is restricted to serial port, but I am not sure how to implement this restriction. Is this OK ?

Thanks again for your time.

Reply:
Ok. Done. Thanks for the suggestion